bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -0,0 +1,436 @@
+            }
+            try {
+                switch (channelIdWithoutGroup) {
+                    case CHANNEL_LOADPOINT_MODE:
+                        evccAPI.setMode(loadpoint, command.toString());
+                        break;
+                    case CHANNEL_LOADPOINT_MIN_SOC:
+                        evccAPI.setMinSoC(loadpoint, Integer.parseInt(command.toString().replaceAll(" %", "")));
+                        break;
+                    case CHANNEL_LOADPOINT_TARGET_SOC:
+                        evccAPI.setTargetSoC(loadpoint, Integer.parseInt(command.toString().replaceAll(" %", "")));
+                        break;
+                    case CHANNEL_LOADPOINT_TARGET_TIME:
+                        if (targetTimeEnabled) {
+                            try {
+                                targetTimeZDT = ZonedDateTime.parse(command.toString(),
+                                        DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ")); // RFC 922 timezone
+                                evccAPI.setTargetCharge(loadpoint, targetSoC, targetTimeZDT);
+                                ChannelUID channel = new ChannelUID(getThing().getUID(), "loadpoint" + loadpoint,
+                                        CHANNEL_LOADPOINT_TARGET_TIME);
+                                updateState(channel, new DateTimeType(targetTimeZDT));
+                            } catch (DateTimeParseException e) {
+                                logger.debug("Failed to set target charge", e);
+                            }
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_TARGET_TIME_ENABLED:
+                        if (command == OnOffType.ON) {
+                            evccAPI.setTargetCharge(loadpoint, targetSoC, targetTimeZDT);
+                            targetTimeEnabled = true;
+                        } else {
+                            evccAPI.unsetTargetCharge(loadpoint);
+                            targetTimeEnabled = false;
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_PHASES:
+                        evccAPI.setPhases(loadpoint, Integer.parseInt(command.toString()));
+                        break;
+                    case CHANNEL_LOADPOINT_MIN_CURRENT:
+                        evccAPI.setMinCurrent(loadpoint, Integer.parseInt(command.toString().replaceAll(" A", "")));
+                        break;
+                    case CHANNEL_LOADPOINT_MAX_CURRENT:
+                        evccAPI.setMaxCurrent(loadpoint, Integer.parseInt(command.toString().replaceAll(" A", "")));
+                        break;
+                    default:
+                        return;
+                }
+            } catch (EvccApiException e) {
+                Throwable cause = e.getCause();
+                if (cause == null) {
+                    logger.debug("Failed to command channel {}: {}", channelUID, e.getMessage());
```

Comment:
Probably some rephrasing needed here, perhaps "Failed to handle command for channel"?